packages/common/src/templates/gatsby.ts

```diff
@@ -1,8 +1,39 @@
-import Template from './template';
+import Template, { ViewConfig } from './template';
 import { decorateSelector } from '../theme';
 import configurations from './configuration';
 
-export default new Template(
+class GatsbyTemplate extends Template {
+  getViews(): ViewConfig[] {
+    const GATSBY_VIEWS: ViewConfig[] = [
+      {
+        views: [
+          { id: 'codesandbox.browser' },
+          {
+            id: 'codesandbox.browser',
+            closeable: true,
+            options: {
+              port: 8080,
```

Comment:
Is the devtools always on port `8080`? If someone would choose to host the main Gatsby app on port 3000, would it still work?

Reply:
If the devtools are always on the same port as the main app, it may be smart to just remove the `port` option, so it will default to being hosted on the main port.

Reply:
ohhh, I didnt know you could use without the port option

done